openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -535,6 +537,7 @@ The technology described in this specification was made available from contribut
    * Require encrypted responses
    * Remove reference to `client_id_scheme` parameter that no longer exists in OpenID4VP
    * Refresh tokens are now optional
+   * Add specific requirements for response encryption
```

Comment:
Should be in -02 I do believe?

Reply:
You are correct. I only went so far as to look at the draft version in the head of the source, which was -01, but didn't look at https://openid.net/specs/openid4vc-high-assurance-interoperability-profile-1_0-01.html

Reply:
and 45c5a6eca1dfde99dc139b3c548ff111fd3f03bd fixes that in this PR